diesel_cli/src/migrations/diff_schema.rs

```diff
@@ -89,7 +86,19 @@ pub fn generate_sql_based_on_diff_schema(
     }
 
     let mut schema_diff = Vec::new();
-
+    let table_names = load_table_names(&mut conn, None)?;
+    let tables_from_database = filter_table_names(
+        table_names.clone(),
+        &config
+            .print_schema
+            .all_configs
+            .first_key_value()
+            .ok_or(errors::Error::UnsupportedFeature(
+                "select exact one print schema key".to_string(),
```

Comment:
This should default to the default schema if no `print-schema-key` is set explicitly.